tests/setup_machine.sh

```diff
@@ -226,12 +396,17 @@ if ! grep '^[[:blank:]]*user_allow_other' /etc/fuse.conf >/dev/null; then
 	echo "user_allow_other" >> /etc/fuse.conf
 fi
 
+echo; echo Configure local port range for outbond connections
+if [ ! -f  /etc/sysctl.d/60-ip_port_range.conf ]; then
+	echo "net.ipv4.ip_local_port_range=10000 64000" > /etc/sysctl.d/60-ip_port_range.conf
+fi
+
```

Comment:
Actually, now that I think about, doesn't this only apply to Ubuntu?

Reply:
> no

I was worried it was because generally I have only had to interact with them in Ubuntu/Debian systems. But yeah, you're right.